x/mock/simulation/doc.go

```diff
@@ -2,26 +2,25 @@
 Package simulation implements a simulation framework for any state machine
 built on the SDK which utilizes auth.
 
-It is primarily intended for fuzz testing the integration of modules.
-It will test that the provided operations are interoperable,
-and that the desired invariants hold.
-It can additionally be used to detect what the performance benchmarks in the
-system are, by using benchmarking mode and cpu / mem profiling.
-If it detects a failure, it provides the entire log of what was ran,
+It is primarily intended for fuzz testing the integration of modules.  It will
+test that the provided operations are interoperable, and that the desired
+invariants hold.  It can additionally be used to detect what the performance
```

Comment:
Why was this changed? It is standard to make newlines at punctuation marks for markdown.

Reply:
This is just using standard word wrap at 100 characters - both are acceptable afaik, I find word wrap easier to read in paragraph form like this

Reply:
I think there's a few double spaces in there btw.